### PR TITLE
DE43118 - Encode orgUnitId replace strings from ISF on content insert and decode them on post-process

### PR DESCRIPTION
**Pull request by grant-cleary**
The old editor did this by hooking scripts from a custom plugin into the editor's `SetContent` and `PostProcess` events. Because we can embed this in ISF, we can do the encoding directly in the ISF return, though decoding still needs to be done via the `PostProcess` event.

### CODE REVIEW

File: components/plugins/isf.js [hunk 1]
```diff
@@ -376,12 +404,37 @@ export function addISFPlugin() {
 			}
 		};
 
+		const decodeReplaceStrings = (html) => {
+			const decode = (match) => {
+				const tempNode = new DOMParser().parseFromString(match, 'text/html');
+
+				const img = tempNode.firstChild;
```
Comment (grant-cleary) [inline components/plugins/isf.js:411]: Whoops, I missed changing this in the way the `encode` method was changed. This will never return anything useful.

Comment (grant-cleary) [reply components/plugins/isf.js]: I'll also push a change shortly to return early if the document body doesn't exist or has no children.

**Review by dbatiste — APPROVED** [review]

File: components/plugins/isf.js [hunk 2]
```diff
@@ -35,6 +35,35 @@ export function addISFPlugin() {
 		const component = context.component;
 		const wmodeOpaque = context.wmodeOpaque;
 
+		const encodeReplaceStrings = (html) => {
```
Comment (grant-cleary) [inline components/plugins/isf.js:38]: Damn it. I accidentally wiped out these changes.

File: components/toolbar/toolbar-inline-limited.js [hunk 3]
```diff
@@ -32,7 +32,7 @@ export function getPlugins(component) {
 		'emoticons',
 		'link',
 		'lists',
-		component.mentions && D2L.LP ? 'mentions' : '',
+		component.mentions && component._context ? 'mentions' : '',
```
Comment (grant-cleary) [inline components/toolbar/toolbar-inline-limited.js:35]: Missed this when I switched it over in the full toolbar.

**Review by dbatiste — APPROVED** [review]

**Pull request merged**